sql/pre_install/tables.sql

```diff
@@ -207,6 +207,46 @@ CREATE INDEX chunk_index_hypertable_id_hypertable_index_name_idx ON _timescaledb
 
 SELECT pg_catalog.pg_extension_config_dump('_timescaledb_catalog.chunk_index', '');
 
+-- Track statistics for columns of chunks from a hypertable.
+-- Currently, we track the min/max range for a given column across chunks.
+-- More statistics (like bloom filters) can be added in the future.
+--
+-- A "special" entry for a column with invalid chunk_id, PG_INT64_MAX,
+-- PG_INT64_MIN indicates that min/max ranges could be computed for this column
+-- for chunks.
+--
+-- The ranges can overlap across chunks. The values could be out-of-date if
+-- modifications/changes occur in the corresponding chunk and such entries
+-- should be marked as "invalid" to ensure that the chunk is in
+-- appropriate state to be able to use these values. Thus these entries
+-- are different from dimension_slice which is used for tracking partitioning
+-- column ranges which have different characteristics.
+--
+-- Currently this catalog supports datatypes like INT, SERIAL, BIGSERIAL,
+-- DATE, TIMESTAMP etc. by storing the ranges in bigint columns. In the
+-- future, we could support additional datatypes (which support btree style
+-- >, <, = comparators) by storing their textual representation.
+--
+CREATE TABLE _timescaledb_catalog.chunk_column_stats (
+  id serial NOT NULL,
```

Comment:
There's a function `ts_chunk_column_stats_update_by_id` which uses this `id` field to update an existing entry. Otherwise every lookup will have to do a `ht_id, chunk_id, col_name` search.

Reply:
Is that function commonly used? If not, is seems to make more sense to use the composite key as the primary key. It makes sense to have an id column if you have a foreign key to the table or if the id is needed to make otherwise similar rows unique (say, being able to store multiple rows with (hypertable, chunk, colname)) but that does not seem to be the case here.